python/ray/worker.py

```diff
@@ -228,7 +228,7 @@ def set_mode(self, mode):
     def set_load_code_from_local(self, load_code_from_local):
         self._load_code_from_local = load_code_from_local
 
-    def put_object(self, value, object_ref=None, pin_object=True):
+    def put_object(self, value, object_ref=None):
```

Comment:
Given that creating an unpinned primary (and only) copy of an object doesn't fit with the object [memory management model](https://docs.google.com/document/d/1lAy0Owi-vPz2jEqBSaHNQcy2IBSDEHyXNOQZlGuj93c/preview#heading=h.61xcnifjkb6v), I thought that it'd be better if creating such an unpinned object was exposed in a testing utility rather than through the externally facing API.

Reply:
Makes sense!